infertrade/algos/community/allocations.py

```diff
@@ -941,6 +941,48 @@ def vortex_strategy(df: pd.DataFrame, window: int = 14, max_investment: float =
     return df
 
 
+def MACDADX_Startegy(df: pd.DataFrame, window_slow: int = 26, window_fast: int = 12, window_signal: int = 9, window_adx: int =14,
```

Comment:
Typo in name - should be strategy I believe?

Reply:
Fixed typo